e2e/__tests__/__snapshots__/test-todo.test.js.snap

```diff
@@ -0,0 +1,82 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`shows error messages when called with invalid argument 1`] = `
+"FAIL __tests__/todo_non_string.test.js
+  ● Test suite failed to run
+
+    Invalid first argument: () => {}. Todo must be called with a string.
```

Comment:
This might look weird, when the fn is not a oneliner.

Reply:
True, perhaps we should just remove this as per @rickhanlonii's comments below. I'm pretty flexible with how this API looks. Maybe just validation of a minimum one arg with the first being a string? :)

Reply:
I think just a simple "todo must be called with just a string", or something similar, is better. No need  to stringify the implementation as long as our stack trace point points back to the declaration